client/layout/guided-tours/tours/plugins-basic-tour.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { makeTour, Tour, Step, ButtonRow, Quit } from 'layout/guided-tours/config-elements';
+import { isEnabled } from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+import { getSelectedSite } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import { abtest } from 'lib/abtest';
+
+const isAtomic = state => {
+	const selectedSite = getSelectedSite( state );
+	if ( ! selectedSite ) {
+		return false;
+	}
+
+	return isSiteAutomatedTransfer( state, selectedSite.ID );
+};
+
+const isABTesting = () => abtest( 'calypsoifyPlugins' ) === 'pointToWPAdmin';
```

Comment:
Let's replace this with a normal feature flag, in case we need to disable it quickly.